.github/workflows/android-check-metadata.yaml

```diff
@@ -4,8 +4,8 @@ on:
   pull_request:
     paths:
       - .github/workflows/android-check-metadata.yaml  # Run check on self change
-      - android/src/fdroid/**
-      - android/src/google/**
+      - android/app/src/main/fdroid/**
+      - android/app/src/main/google/**
```

Comment:
Can we keep store descriptions in their old locations? It's not a code. Or please check the documentation if they can be moved somehow out of the app and src folders.

Reply:
yeah probably I'll look into it

Reply:
Well the old android/src location does not exist anymore and there is no sense in keeping it. If you don't like the current new location, we can find a new one like `android/app/stores`

Reply:
Can stores info be stored in separate module?

Reply:
its needed in the :app module, so it makes sense to store it in that module, but not necessarily in the /src folder

Reply:
What is the connection between building the app and uploading texts/screenshots into a store?

Reply:
there is some logic inside the app's build.gradle file to upload the app to the play store. This logic uses the release notes from those folders, so it makes sense to keep them inside the app module

Reply:
This will break F-Droid.

Reply:
https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/#triple-t-structure

> The second supported structure is compatible with the tool [Triple-T Gradle Play Publisher](https://github.com/Triple-T/gradle-play-publisher/blob/2.8.0/README.md#publishing-listings).

> Place the following structure at <module>/src/main/play/ or, for build flavor support, at <module>/src/<buildFlavor>/play/, where <module> is the gradle module of your app (in many cases, this is just the folder app):The second supported structure is compatible with the tool [Triple-T Gradle Play Publisher](https://github.com/Triple-T/gradle-play-publisher/blob/2.8.0/README.md#publishing-listings).

> Place the following structure at <module>/src/main/play/ or, for build flavor support, at <module>/src/<buildFlavor>/play/, where <module> is the gradle module of your app (in many cases, this is just the folder app):

probably should work.

Reply:
right now it is in <module>/src/main/fdroid/play. Do you think it will work? Can we merge fdroid and google play listings in a single <module>/src/main/play folder?

Reply:
No, we can not merge them. Listings will differ in the future.